libs/community/langchain_community/vectorstores/surrealdb.py

```diff
@@ -202,14 +208,19 @@ async def _delete(ids: Optional[List[str]], **kwargs: Any) -> Optional[bool]:
         return asyncio.run(_delete(ids, **kwargs))
 
     async def _asimilarity_search_by_vector_with_score(
-        self, embedding: List[float], k: int = 4, **kwargs: Any
+        self,
+        embedding: List[float],
+        k: int = DEFAULT_K,
+        filter: Optional[Dict[str, str]] = None,
+        **kwargs: Any,
     ) -> List[Tuple[Document, float]]:
```

Comment:
this return type is no longer valid

Reply:
Many thanks to both of you @hwchase17  and @baskaryan :)